src/color_picker_cell.coffee

```diff
@@ -0,0 +1,25 @@
+###*
+ * ColorPickerCell
+ *
+ * This is an internal-use view to control the logic of a color cell in
+ * the color picker.
+ * @class
+ * @augments {Ember.View, Ember.Widgets.StyleBindingMixin,
+ *            Ember.Widgets.ColorPickerMixin}
+###
+Ember.Widgets.ColorPickerCellComponent =
```

Comment:
it's good to also comment on what needs to be passed in, in order to use this component, e.g. `color`